app/models/patient_changeset.rb

```diff
@@ -30,22 +31,49 @@
 #  fk_rails_...  (school_id => locations.id)
 #
 class PatientChangeset < ApplicationRecord
-  attribute :pending_changes,
+  attribute :data,
             :jsonb,
             default: {
-              child: {
+              upload: {
+                child: {
+                },
+                parent_1: {
+                },
+                parent_2: {
+                },
+                academic_year: nil,
+                home_educated: nil,
+                school_move_source: nil
               },
-              parent_1: {
-              },
-              parent_2: {
-              },
-              search_results: []
+              search_results: [],
+              review: {
+                patient: {
+                },
+                school_move: {
+                }
+              }
             }
+
   belongs_to :import, polymorphic: true
   belongs_to :patient, optional: true
   belongs_to :school, class_name: "Location", optional: true
 
-  enum :status, { pending: 0, processed: 1 }, validate: true
+  enum :status,
+       {
+         pending: 0,
+         processed: 1,
+         calculating_review: 2,
+         ready_for_review: 3,
+         import_invalid: 4,
+         committing: 5,
+         needs_re_review: 6,
+         cancelled: 7
```

Comment:
Just a note ... and it may not be worth changing now, but I think I would've added these as necessary in the PRs to come so that it's clear where they're needed and used.